appcues/src/main/java/com/appcues/ui/AppcuesOverlayViewManager.kt

```diff
@@ -176,14 +176,14 @@ internal fun Activity.getParentView(): ViewGroup {
     // dialog. In this case, we need to apply the fix-ups below to ensure that our
     // content can render correctly inside of this other view. In each case, we use
     // the applicable value from the Activity default window.
-    if (ViewTreeLifecycleOwner.get(decorView) == null) {
+    if (decorView.findViewTreeLifecycleOwner() == null) {
         val lifecycleOwner = window.decorView.findViewTreeLifecycleOwner()
-        ViewTreeLifecycleOwner.set(decorView, lifecycleOwner)
+        decorView.setViewTreeLifecycleOwner(lifecycleOwner)
     }
 
-    if (ViewTreeViewModelStoreOwner.get(decorView) == null) {
+    if (decorView.findViewTreeViewModelStoreOwner() == null) {
         val viewModelStoreOwner = window.decorView.findViewTreeViewModelStoreOwner()
-        ViewTreeViewModelStoreOwner.set(decorView, viewModelStoreOwner)
+        decorView.setViewTreeViewModelStoreOwner(viewModelStoreOwner)
```

Comment:
this was an API change in a dependency that comes in through Compose somehow it seems - these are now functions on View rather than static functions on the classes.